### PR TITLE
[C-5501] Alexander/c 5501 Test send_message with timeout prop

### PR DESCRIPTION
**Pull request by lednax**
## Description of the change

> Description here
- Adds test for new timeout prop
- Adds example

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]()

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by vatasha** [comment]
LGTM although this is redundant :)

**Review by cgradwohl — COMMENTED** [review]
what is this file for? I am confused by this method and file.

File: v2/examples/send/send_message_timeout.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package examples
+
+import (
+	"context"
+	"log"
+
+	"github.com/trycourier/courier-go/v2"
+)
+
+func sendMessage() {
```
Comment (cgradwohl) [inline v2/examples/send/send_message_timeout.go:10]: doesn't this already exist? why are we making another send method?

Comment (lednax) [reply v2/examples/send/send_message_timeout.go:10]: It's an example. There's no impact here.

**Review by cgradwohl — APPROVED** [review]

**Pull request merged**